### PR TITLE
frobenius_norm onnx export added

### PR DESCRIPTION
**Pull request by neginraoof**

### CODE REVIEW

**Comment by neginraoof** [comment]
@bddppq please review

File: test/onnx/test_onnx_opset.py [hunk 1]
```diff
@@ -393,5 +392,31 @@ def forward(self, x):
 
         check_onnx_opsets_operator(MyModule(), x, ops, opset_versions=[9, 10])
 
+    def test_frobenius_norm(self):
```
Comment (lara-hdr) [inline test/onnx/test_onnx_opset.py]: We are now keeping test_onnx_opset.py for testing special cases only. 
No need to add tests here in this PR.

**Comment by lara-hdr** [comment]
looks good :)

**Review by bddppq — APPROVED** [review]
Will land once the review comment in test_onnx_opset.py is resolved

**Comment by neginraoof** [comment]
@bddppq this is updated and ready

**Comment by VitalyFedyunin** [comment]
Phabricator tests are failing with onnx errors.

**Comment by neginraoof** [comment]
@VitalyFedyunin @bddppq I don't have access. Where can I find more details?

**Comment by neginraoof** [comment]
@VitalyFedyunin @bddppq  Could you help with this?

**Comment by bddppq** [comment]
Looking

**Comment by bddppq** [comment]
It was a flaky test, landing

**Comment by neginraoof** [comment]
@bddppq Any updates on this PR?

**Pull request closed without merging**